ixr7220h4-32d/modules/fpga.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2014 Broadcom Corporation
```

Comment:
Insert our copyright above the bcm one. Add 

Copyright (C) 2024 Nokia

do this for all other instances of new code we are publishing.